plane/source/docs/rtl-mode.rst

```diff
@@ -4,28 +4,33 @@
 RTL Mode (Return To Launch)
 ===========================
 
-In RTL mode the plane will return to its home location (the point where the
-plane armed - assuming it had GPS lock) and loiter there until given
-alternate intructions (or it runs out of fuel!). As with :ref:`AUTO <auto-mode>` mode
+Return to Launch mode is intended to bring a plane back to the launch location during a 
+failsafe event or an operator's command. The plane will return to the launch location 
```

Comment:
Sorry, but this moves it from a factually correct (if not marked up correctly) description to something not actual correct.

Typically where you *arm* is where home is set.  If you arm in the pits, taxi out to the runway then takeoff - and don't set a rally point, reset your home or have `DO_LAND_START` set, you may really annoy people when your vehicle is circling above the pits.  The vehicle will not return to the "launch point" - the runway - but to the "home location".

The words "home location" should really be marked up - it's a real thing in the ArduPilot code.  "launch location" is not.

Reply:
The only reason I use "Launch" was because it's called RTL - NOT RTH. Maybe that is what needs changing.